app/src/main/java/com/nextcloud/talk/controllers/ChatController.kt

```diff
@@ -306,6 +309,9 @@ class ChatController(args: Bundle) :
         }
 
         this.voiceOnly = args.getBoolean(BundleKeys.KEY_CALL_VOICE_ONLY, false)
+
+        hasChatPermission =
+            AttendeePermissionsUtil(currentConversation!!.permissions).hasChatPermission(conversationUser)
```

Comment:
Why is this initialized both here and in `getRoomInfo()`? Just trying to understand the flow of this controller

Reply:
well at first i only initialized in `init`. 
Then i added it also to getRoomInfo() to upadte the UI on the fly when permissions on server were changed.

i was also thinking about to remove it from `init`. This should be okay and work, but i was not 100% sure if somehow hasChatPermission can be accessed before getRoomInfo()...

Reply:
well, in fact it crashed when opening chat from notification when it's done in init..
https://github.com/nextcloud/talk-android/pull/2058